File: Code Chef/JavaScript/300-399/399 The Cheaper Cab CABS.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    return intArray;
};

const whichCabToTake = (cab1charges, cab2charges) => {
    (cab1charges === cab2charges)
        ? console.log("ANY")
        : (cab1charges < cab2charges)
            ? console.log("FIRST")
            : console.log("SECOND");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n").slice(1));
    
    testcases.forEach((testcase) => {
        const [cab1charges, cab2charges] = testcase;
        whichCabToTake(cab1charges, cab2charges);
    });
});

File: Code Chef/C Programming/300-399/399 The Cheaper Cab CABS.c
#include <stdio.h>
int main()
{
    int cases,a,b;
    scanf("%d",&cases);
    for (int i=0;i<cases;i++){
        scanf("%d %d",&a,&b);
        if(a<b){
            printf("FIRST\n");
        }
        else if(b<a){
            printf("SECOND\n");
        }
        if (a==b){
            printf("ANY\n");
        }
    }
    return 0;
}